bin/ch/WScriptJsrt.cpp

```diff
@@ -390,6 +395,14 @@ JsErrorCode WScriptJsrt::LoadModuleFromString(LPCSTR fileName, LPCSTR fileConten
 
     // ParseModuleSource is sync, while additional fetch & evaluation are async.
     unsigned int fileContentLength = (fileContent == nullptr) ? 0 : (unsigned int)strlen(fileContent);
+ 
+    if(isFile && fullName)
+    {
+        JsValueRef moduleURL;
+        ChakraRTInterface::JsCreateString(fullName, strlen(fullName), &moduleURL);
+        errorCode = ChakraRTInterface::JsSetModuleHostInfo(requestModule, JsModuleHostInfo_ModuleURL, moduleURL);
```

Comment:
This errorCode is lost, right? Seems we should check it directly or use `IfJsrtErrorSetGo`.